vagrant/provisioning/roles/upgrade-solr/tasks/main.yml

```diff
@@ -0,0 +1,91 @@
+- name: check for available disk space
+  become: yes
+  command: df -h /opt/
+  register: space_available
+
+- name: check for solr data space
+  become: yes
+  stat:
+    path:  /opt/arkcase/data/solr
+  register: solr_data
+
+- name: upgrade and backup solr if space is available under root_folder
+  block: 
+    - name: stop arkcase
+      become: yes
+      systemd:
+        name: arkcase
+        state: stopped
+
+    - name: copy existing solr data to backup folder
```

Comment:
first stop Solr, then backup the data and delete the cores, right?

Reply:
No. We can't delete the cores if Solr is not running. So, backup data, delete cores, stop Solr.